src/transaction/types.ts

```diff
@@ -27,10 +29,15 @@ export type _Input = {
   outputIndex: number,
 }
 
+export type _Asset = Map<Buffer, BigInt>
+
+export type _MultiAsset = Map<Buffer, _Asset>
+
 export type _Output = {
   address: Buffer,
-  coins: Lovelace,
+  coins: Lovelace | [Lovelace, _MultiAsset],
```

Comment:
Pretty late comment I know, but would it work better to parse outputs into 
```
{
address: Buffer,
coins: Lovelace,
tokenBundle: _MultiAsset
}
and make it a bit easier for us when we are preparing the outputs for the hw wallets?

Reply:
Fixed